docs/cep/reference/extensions/index.md

```diff
@@ -0,0 +1,86 @@
+---
+sidebar_position: 3
+title: Extensions
+---
+
+# Extensions
+
+Extensions enhances Macrometa Stream QL by incorporating additional functionalities in a seamless manner.
+
```

Comment:
This would be a good place to explain to the user why they would want to use extensions, like a few high-level use cases. Something like:

You can use extensions to create stream workers that  (do whatever) and output the results to (format that makes sense).

Reply:
@diana-macrometa - The `CEP` has core and extensions. The core remains almost invisible for the user except the language and some functions everything else related to `sources` and `sinks` are extensions.

Reply:
That is helpful. Can you explain to our theoretical noob user how extensions are cool and why they should use them?

Reply:
@diana-macrometa - I'm wandering whether `extension` is the correct term since we neither allow customers to create their own nor get such from e.g. `store`?

Reply:
I do not know. @durgagokina once said the idea of "extensions" should go away. Whatever we call them, we still need to explain to users how to use these libraries and help them understand their potential. Once you understand what they can do, it's really cool stuff.